phpBB/styles/prosilver/template/search_body.html

```diff
@@ -91,7 +101,9 @@ <h2 class="solo">{L_SEARCH}</h2>
 </div>
 
 </form>
+<!-- EVENT search_body_form_after -->
 
+<!-- EVENT search_body_recent_search_before -->
```

Comment:
In theory, to display some custom info unrelated to recent searches (or f.e. custom recent search loop).